src/System.Security.Cryptography.Xml/tests/Resources/System.Security.Cryptography.Xml.Tests.rd.xml

```diff
@@ -0,0 +1,13 @@
+<Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
+  <Library>
+
+    <!-- These types are loaded through CryptoConfig -->
+    <Type Name="System.Security.Cryptography.RSAPKCS1SignatureFormatter" Dynamic="Required Public" />
```

Comment:
@bartonjs , any particular reason these aren't in the CryptoConfig hashtable? That would probably fix this.